src/routes/(app)/org/[id=integer]/+page.svelte

```diff
@@ -48,6 +48,45 @@
 		});
 	};
 
+	const addShareModel = () => {
+		addToast({
+			type: 'success',
+			message: 'Sharing is caring!',
+			life: 5000
+		});
+
+		window.removeEventListener('mousedown', addShareModel);
+		window.removeEventListener('focus', addShareModel);
+	};
+
+	const startShare = async () => {
+		let shareUrlId = data.org.slug?.slug || data.org.id;
+		let shareUrl = `${window.origin}/org/${shareUrlId}`;
+
+		try {
+			await navigator.share({
+				url: shareUrl
+			});
+			window.addEventListener('mousedown', addShareModel);
+			window.addEventListener('focus', addShareModel);
```

Comment:
Are you sure this wont run if a person navigates to a different page on their history?

Reply:
Based on my current testing, I do not believe it will.